packages/manager/apps/vrack-services/src/components/vrack-id.component.tsx

```diff
@@ -60,6 +60,17 @@ export const VrackId: React.FC<VrackServicesWithIAM & {
         );
       },
     },
+    {
+      id: 2,
+      label: tDashboard('vrackActionAssociateToAnother'),
+      onClick: () => {
+        navigate(
+          urls.overviewAssociateAnother
+            .replace(':id', vs.id)
+            .replace(':vrackId', vs.currentState.vrackId),
```

Comment:
So that way of adding params is not deprecated finally, I thought it was

Reply:
I have no idea